app/dashboard/_components/cards/InclusionScore/InclusionScoreChart.tsx

```diff
@@ -0,0 +1,94 @@
+'use client'
+import {
+  AreaChart,
+  Area,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  ResponsiveContainer,
+} from 'recharts'
+
+type DataItem = {
+  month: string
+  first_day_of_week: number
+  iso_week: string
+  team: string
+  inclusion_score: number
+  work_habits: number
+  cross_functional_inclusion: number
+  informal_influence: number
+  benchmark: number
+  [key: string]: number | string
+}
+
+type AggregatedDataEntry = {
+  month: string
+  [team: string]: number | string | null
+  benchmark: number
+}
+
+type ChartComponentProps = {
+  data: DataItem[] | AggregatedDataEntry[]
+  selectedTeam: string
+}
+
+const InclusionScoreChart: React.FC<ChartComponentProps> = ({
```

Comment:
please type the props as in all other components

Reply:
My bad had 3 different versions of the chart, i must have mixed up this. 
Changed it